src/main/resources/db/migration/V20231003__Create_initial_entity_tables.sql

```diff
@@ -429,14 +430,13 @@ CREATE TABLE `moderation_logs`
   `locked`               TINYINT                                   NULL,
   `banned`               TINYINT                                   NULL,
   `featured`             TINYINT                                   NULL,
+  `featured_community`   TINYINT                                   NULL,
   `expires`              TIMESTAMP(3)                              NULL,
   `created_at`           TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP(3) NOT NULL
 ) ENGINE = InnoDB
   DEFAULT CHARSET `utf8mb4`
   COLLATE = 'utf8mb4_unicode_ci';
 
-CREATE UNIQUE INDEX `IDX_MODERATION_LOGS_INSTANCE_ID_ACTION_TYPE_ENTITY_ID` ON `moderation_logs` (`instance_id`, `action_type`, `entity_id`);
```

Comment:
Why are you removing the index?

Reply:
We can still create an index, just not unique. Most mod actions can be undone which creates a new modlog for the same action type where one of the TINYINT columns is inverted.

Reply:
Good point, I think we should add some indexes that reflect the query to help improve lookups.

Reply:
Added indexes and foreign keys